packages/orbit-components/src/utils/Grid/README.md

```diff
@@ -75,53 +73,13 @@ All of the object for these properties have the same properties and none is requ
 
 ## Why you should use this component
 
-By default, `styled-components` does not auto-prefix any CSS grid properties, so you would end up auto-prefixing on your own.
-Also, if you want to support IE 10+, you have to use only the old specification, which has a lot of limitations and does not provide the flexibility of modern specification.
-
 The main focus of this component is to provide an API that solves the most common issues and gives you a quick how to start using CSS grids in your project without worries.
 
 By default, this Grid component handles:
 
 - Compiling the `repeat` function to the old specification
 - Auto-placement of its children
 
-## Known template limitations
```

Comment:
Will that allow us to do things more efficiently in the codebase? i.e. are there some further improvements we should keep track of in Jira?

Reply:
I have removed these because they referred to IE limitations. I don't think they're a reality nowadays and we don't support IE officially, so I wouldn't say it makes sense to mention limitations of one component that are exclusive to IE

Reply:
Okay, so there's no more IE-related workaround that's still in the codebase to your knowledge?

Reply:
AFAIK, no